src/checks/graphiteSpike.check.js

```diff
@@ -88,7 +91,7 @@ class GraphiteSpikeCheck extends Check {
 				this.checkOutput = ok ? 'No spike detected in graphite data' : 'Spike detected in graphite data';
 			})
 			.catch(err => {
-				console.error('Failed to get JSON', err);
+				logger.error(`event=${logEventPrefix}_ERROR message=${err.message} stack="${err.stack.replace(/\n/g, '; ')}" url=${this.sampleUrl}`);
```

Comment:
can just pass in the `err` object, logger will take care of extract/sanitising the data

```logger.error({ event: logEventPrefix, url: this.sampleUrl }, err)```